### ISSUE TITLE
[Ubuntu Jaunty]python-numpy install SyntaxWarning (migrated from Trac #1105)

### ISSUE THREAD

**Issue by thouis** [thouis/numpy-trac-migration#2658]
Original ticket http://projects.scipy.org/numpy/ticket/1105
Reported 2009-05-12 by trac user jhaitas, assigned to unknown.

This is reported on Launchpad at: [https://bugs.launchpad.net/ubuntu/+source/python-numpy/+bug/335646 |https://bugs.launchpad.net/ubuntu/+source/python-numpy/+bug/335646]

Setting up python-numpy (1:1.2.1-1ubuntu1) ...
/usr/lib/python2.6/dist-packages/numpy/lib/tests/test_io.py:68: SyntaxWarning: assertion is always true, perhaps remove parentheses?
  assert(c.readlines(),

Install seems to be OK.

**Comment by thouis** [comment]
Attachment in Trac by trac user wprins, 2009-08-15: [issue_1105_fix.diff](http://projects.scipy.org/numpy/attachment/ticket/1105/issue_1105_fix.diff)

**Comment by thouis** [comment]
Comment in Trac by trac user wprins, 2009-08-15

I've been installing Ubuntu Karmic pre-release tonight and noticed this bug crop up during installation.  It's harmless, nevertheless I decided to see if it was already reported and found this report here.  

I've inspected the problem and it appears that simply instead of calling "assert_equal" in one test, instead the "_equal" bit was left off, resulting in the warning message.  

I've thus fixed it trivially by fixing this call.  A patch that demonstrates this trivial fix is attached.
